app/models/course.rb

```diff
@@ -14,7 +12,11 @@ class Course < ApplicationRecord
            after_remove: :touch_tag,
            after_add: :touch_tag
 
-  has_many :media, -> { order(position: :asc) }, as: :teachable
+  # rubocop:todo Rails/HasManyOrHasOneDependent
```

Comment:
I think we discussed that we want to disable this cop, right?

Reply:
Yes, I've just done this in 82c572e39b.